index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+
+    <title>portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="bar1"> 9:41
+            <img src="./images/Group 4.png" alt="icons"></div>
+        <div class="bar2">
+            <h1>TRACY MIRANJA</h1>
+            <nav>
+                <img src="./images/Icon - Menu.png" alt="togglenav">
+            </nav>
+        </div>
+    </header>
+    <div class="container">
+        <div class="banner">
+            <h1>Hey There. I’m Tracy</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul class="social-links">
```

Comment:
- [OPTIONAL] Kindly try as much as you can to replicate the original design. Work on the positioning and spacing of elements.

![image](https://user-images.githubusercontent.com/98453979/205265323-d09ff0a2-568a-4672-bc03-e0899f183aa1.png)
 
### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205265724-47c0ca9c-ee40-411d-9eaa-8f86c74e5c46.png)